tarodb/sqlite/migrations/000007_chain_txns_height_hint.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE chain_txns
```

Comment:
FWIW I think we can just add this directly to the table. We're still in a pretty unstable phase, so people should expect their DB format to just change over night.

Reply:
Done.